src/helpers/sendFirebaseNotification.ts

```diff
@@ -60,18 +61,28 @@ export default async function ({
         },
       }
     }
-    console.log('Send next chunk of tokens by sendMulticast')
     const response = await messaging.sendMulticast(message)
-    response.responses.forEach(async (response) => {
+    const deleteTokens: string[] = []
+    response.responses.forEach(async (response, index) => {
+      const token = chunk[index]
       if (response.success) {
         console.log(response)
         await storeLastTimeSent(Date.now())
         return
       }
       if (!response.error) return
-      if (response.error.code === 'messaging/registration-token-not-registered')
+      const errorCode = response.error.code
+      if (
+        /(registration-token-not-registered|invalid-registration-token|invalid-argument)/.test(
+          errorCode
+        )
+      ) {
+        console.log(errorCode)
```

Comment:
`console.error`, also, we may want to report this somewhere

Reply:
I don't think we need this console here, because we have one on the L.84

Reply:
> I don't think we need this console here, because we have one on the L.84

But in case of certain errors it will not reach it
We have `return` on the L.82